practice-project-ui/src/Calculator.tsx

```diff
@@ -1,9 +1,13 @@
+import FormControl from '@mui/material/FormControl';
```

Comment:
You should be able to group all of these `@mui/material` components by importing them like `{ FormControl, TextField, ....} from '@mui/material'`

Reply:
So it just automatically looks in subdirectories if it doesn't find the import reference in the referenced directory ie 'mui/material' ? Or would it specifically look for a subdirectory with the same name as the import module?